components/ActionBox/components/DealActionBox.tsx

```diff
@@ -0,0 +1,265 @@
+						<InfoClick onClick={() => setShowTooltip(!showTooltip)}>
+							<Image src={Info} alt="info icon" />
+						</InfoClick>
+					</FlexDivRowCentered>
+				</RedemptionPeriodTooltip>
+			</RedemptionHeader>
+			<ContentContainer>
+				<>
+					<ActionBoxInputLabel>{label}</ActionBoxInputLabel>
+					<InputContainer>
+						<ActionBoxInput
+							type={'number'}
+							placeholder={placeholder}
+							value={inputValue}
+							onChange={(e) => {
+								const value = !!e.target.value.length ? parseFloat(e.target.value) : null;
+								setIsMaxValue(false);
+								setInputValue(value);
+							}}
+						/>
+						{maxValue && (
+							<ActionBoxMax
+								isProRata={dealRedemptionData?.status === Status.ProRataRedemption && !isWithdraw}
+								onClick={handleMaxButtonClick}
+							>
+								{dealRedemptionData?.status === Status.ProRataRedemption && !isWithdraw
+									? 'Max Pro Rata'
+									: 'Max'}
+							</ActionBoxMax>
+						)}
+					</InputContainer>
+					<ActionBoxHeaderWrapper>
+						<ActionBoxHeader
+							isAcceptOrReject
+							isPool={false}
+							isSelected={isDealAccept}
+							onClick={() => setIsDealAccept(true)}
+						>
+							<FlexDivCenterRow>
+								<>Accept Deal</>
+								<QuestionMark
+									text={`Choose accept to agree to the deal terms with up to the max amount based on your allocation this round`}
+								/>
+							</FlexDivCenterRow>
+						</ActionBoxHeader>
+						<ActionBoxHeader
+							isAcceptOrReject
+							isPool={false}
+							isSelected={!isDealAccept}
+							isWithdraw={isWithdraw}
+							onClick={() => setIsDealAccept(false)}
+						>
+							<FlexDivCenterRow>
+								<>Withdraw</>
+								<QuestionMark text="Withdraw a portion or all of your capital from the pool and receive your original purchase tokens back" />
+							</FlexDivCenterRow>
+						</ActionBoxHeader>
+					</ActionBoxHeaderWrapper>
+				</>
+			</ContentContainer>
+			{poolId !== '0x7e135d4674406ca8f00f632be5c7a570060c0a15' && (
+				<ActionButton
+					disabled={isDisabled}
+					isWithdraw={isWithdraw}
+					onClick={() => {
+						const setCorrectTxnType = () => {
+							if (isDealAccept) {
+								return setTxType(TransactionType.Accept);
+							}
+							if (isWithdraw) {
+								return setTxType(TransactionType.Withdraw);
+							}
+						};
+						setCorrectTxnType();
+						setShowTxModal(true);
+					}}
+				>
+					{isDealAccept ? 'Accept Deal' : 'Withdraw from Pool'}
+
+					{isPoolDisabled && (
+						<div>
+							<QuestionMark text="Purchasing for this pool has been disabled due to the open period bug on the initial pool factory contracts that has been patched for all new pools moving forward. If you are in this pool we recommend withdrawing at the end of the pool duration. see details here: https://github.com/AelinXYZ/AELIPs/blob/main/content/aelips/aelip-4.md" />
+						</div>
+					)}
+				</ActionButton>
+			)}
+
+			{Number(maxValue ?? 0) < Number(inputValue ?? 0) && (
```

Comment:
maybe this section could be a separate DealActionBoxError component since we are already breaking things up and its a large component

Reply:
+1

Reply:
Done